mapper/src/main/java/io/smallrye/config/mapper/CompareWith.java

```diff
@@ -1,23 +0,0 @@
-package io.smallrye.config.mapper;
-
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
-import java.util.Comparator;
-
-/**
- * Specify the comparator to use to compare the annotated type for range.
- */
-@Documented
-@Retention(RetentionPolicy.RUNTIME)
-@Target(ElementType.TYPE_USE)
-public @interface CompareWith {
```

Comment:
I'd rather implement these than remove them :-)

Reply:
Sure, it was just to reduce the noise and focus on what was actually implemented :)

Reply:
We will implement this in a subsequent PR.